src/components/Navigation/Navigation.tsx

```diff
@@ -1,18 +1,23 @@
 import { useState } from 'react';
-import { useNavigate } from 'react-router-dom';
-import { IconButton, Stack, useMediaQuery, Button } from '@mui/material';
-import { useAuth } from '../Auth/Auth';
-import { getSiteTree } from '../../utils/siteTree';
+import { IconButton, Stack, useMediaQuery, Link } from '@mui/material';
 import Drawer from '../Drawer/Drawer';
+import { Link as RouterLink } from 'react-router-dom';
 import MenuIcon from '@mui/icons-material/Menu';
 import { ToggleDrawerFunction } from '../../types/types';
+import { spacings } from '../DesignSystem/DesignSystem';
 
-const Navigation = () => {
+interface NavigationProps {
+  siteTree: {
+    label: string;
```

Comment:
See my remakrt about the type import, I think we can use it here as well...

Reply:
✅ => 🏷️ ADD sidebar type